examples/00_quick_start/sasrec-tf2.ipynb

```diff
@@ -0,0 +1,321 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.                #    conv_dims = kwargs.get("conv_dims", [100, 100])
</pre><p>I think this guy should be:</p><pre class="ql-syntax" spellcheck="false">conv_dims=[100,100]
</pre><p>because kwargs is not defined here?</p><p>Actually, you may want to add a parameter here at the beginning like the other parameters</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1530/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>